openhands/integrations/github/github_service.py

```diff
@@ -56,14 +66,27 @@ def __init__(
         base_domain: str | None = None,
     ) -> None:
         self.user_id = user_id
+        self.external_auth_id = external_auth_id
         self.external_token_manager = external_token_manager
 
+        # Initialize HTTP client
+        self.github_http_client = GitHubHTTPClient()
```

Comment:
@openhands please redo the instantiation of the github http client; move the logic in the constructor of `GitHubService` to the constructor of `GitHubHTTPClient`